src/openloco/windows/LandscapeGeneration.cpp

```diff
@@ -779,94 +780,94 @@ namespace openloco::ui::windows::LandscapeGeneration
             {
                 case widx::number_of_forests_up:
                 {
-                    *numberOfForests = std::min<int16_t>(*numberOfForests + 10, scenario::max_num_forests);
+                    s5::getOptions().numberOfForests = std::min<int16_t>(s5::getOptions().numberOfForests + 10, scenario::max_num_forests);
                     break;
                 }
                 case widx::number_of_forests_down:
                 {
-                    *numberOfForests = std::max<int16_t>(scenario::min_num_forests, *numberOfForests - 10);
+                    s5::getOptions().numberOfForests = std::max<int16_t>(scenario::min_num_forests, s5::getOptions().numberOfForests - 10);
                     break;
                 }
                 case widx::min_forest_radius_up:
                 {
-                    *minForestRadius = std::min<int16_t>(*minForestRadius + 1, scenario::max_forest_radius);
-                    if (*minForestRadius > *maxForestRadius)
-                        *maxForestRadius = *minForestRadius;
+                    s5::getOptions().minForestRadius = std::min<int16_t>(s5::getOptions().minForestRadius + 1, scenario::max_forest_radius);
```

Comment:
I think the code would be clearer if there were one `options` variable. (One reference in each method or even namespace global, that is.)